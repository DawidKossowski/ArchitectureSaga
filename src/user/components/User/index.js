import React, {Component} from 'react';

export default class User extends Component {
    render() {
        const {data} = this.props;
        return (
          <li>
              <span>{data['first_name']} </span>
              <span>{data['last_name']}</span>
          </li>
        )
    }
}
