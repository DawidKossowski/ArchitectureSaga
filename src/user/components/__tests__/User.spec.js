import React from 'react';
import renderer from 'react-test-renderer'
import User from '../User';

describe('user sagas', () => {
    const props = {
        data: {
            first_name: 'D',
            last_name: 'K',
        }
    };
    it('should fetch users', () => {
        const user = renderer.create(<User {...props} />).toJSON();
        expect(user).toMatchSnapshot();
    });
});
