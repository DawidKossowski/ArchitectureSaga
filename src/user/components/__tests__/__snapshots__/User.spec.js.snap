// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user sagas should fetch users 1`] = `
<li>
  <span>
    D
     
  </span>
  <span>
    K
  </span>
</li>
`;
